### ISSUE TITLE
Could a batchSize parameter be added to insertTable?

### ISSUE THREAD

**Issue by katy-sadowski** [OHDSI/DatabaseConnector#297]
Would it be possible to add a `batchSize` parameter (like the one in `dbWriteTable`) to `insertTable`?  

Use case: I'd like to provide the option to specify a `tempEmulationSchema`, _and_ to insert data in batches.

**Comment by schuemie** [comment]
The aim of the `insertTable()` function is to work with as little user hand-holding across all supported platforms. It currently uses a [hard-coded batch size of 10,000 rows](https://github.com/OHDSI/DatabaseConnector/blob/main/R/InsertTable.R#L336). Is this not working for you?

**Comment by katy-sadowski** [comment]
ah!  i missed that!  that works for me, though maybe couldn't hurt to add it as a param with default value 10000 for more flexibility/clarity?

**Comment by schuemie** [comment]
Well, the behavior can actually be different depending on the database platform, and of course it is a whole different story when you enable bulk loading. The `insertTable()` function should just be smart enough to figure out how to upload your data. If you're running into cases where it's not, let me know and we can fix the behavior.

**Comment by katy-sadowski** [comment]
makes sense. it worked like a charm as-is in Snowflake - i hadn't even tried it before because i thought it wasn't possible to specify the batch size and assumed the default was 1 😂 🤦
